client/templates/minutes/minutesedit.html

```diff
@@ -147,6 +147,10 @@ <h2 id="minutesHeading" class="header">
                         {{#if nextMinutes}}
                             Next: <a id="btnNextMinutesNavigation" href="{{minutesPath nextMinutes._id}}">{{nextMinutes.date}}</a>
                         {{/if}}
+                        &emsp;
+                        {{#if isFinalized}}
```

Comment:
Unfortunately this code does not fulfile multiple acceptance criteria.
Please double check what is needed here.
https://github.com/4minitz/4minitz/issues/448